### DISCUSSION TITLE
Prometheus Rule load via helm chart ?

### DISCUSSION THREAD

**Discussion by dhrumilpatel** [grafana/mimir#3718] [Help and support] [answered]
Hi,

For our single point of monitoring cluster observability and alerting, we decided to using Mimir over Thanos. However, our experience is not a smoot with documentation configurations specifically for kuberntes.

Ex: mimirtool is avaiable for kubernetes ? or alternative method to load custom prometheus rules via helm chart ? ( could not find any consolidated doc, and hence the struggle ) can you suggest alternative, examples or more more clear reference documentations ?

**Comment by dimitarvdimitrov** [comment, marked as answer]
you can use `mimirtool` with any installation of mimir, just point it to the URL of mimir. But there is no kubernetes-native way to configure rules and alerts, such as CRDs. There's an open discussion about CRDs in #2133
